db-tests/src/test/java/gov/nasa/jpl/aerie/database/ExternalEventTests.java

```diff
@@ -0,0 +1,1627 @@
+package gov.nasa.jpl.aerie.database;
+
+import org.junit.jupiter.api.*;
```

Comment:
The style in these tests is to import the specific needed classes rather than `api.*`

```suggestion
import org.junit.jupiter.api.AfterAll;
import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeAll;
import org.junit.jupiter.api.Nested;
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.TestInstance;
```